Shared/src/Datahub.Shared/Entities/PostgresTier.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Datahub.Shared.Entities
+{
+    public class PostgresTier
+    {
+        public string SKUName { get; set; }
+        public int Cores { get; set; }
+        public string MemorySize { get; set; }
+        public string PSQL_SKU { get; set; }
+
+        public string Cost { get; set; }
+
+        public bool IsAvailable { get; set; }
+
+        public static List<PostgresTier> GetPostgresTiers()
+        {
+            var postgresTiers = new List<PostgresTier>();
+            postgresTiers.Add(new PostgresTier { SKUName = "B1ms (default)", Cores = 1, MemorySize = "2GiB", PSQL_SKU = "B_Standard_B1ms", Cost = "$12.41/month", IsAvailable = true });
```

Comment:
we should look at moving this out of the code

Reply:
Actually, before anything else more fancy, can we look at consolidating this with https://github.com/ssc-sp/datahub-portal/pull/1546/files#diff-59c8519459e889066abfe34df3151bcd1481797c2002a1241e03ded06afad55e? @gcnabeel @vb-ssc

Reply:
I will consolidate these in the next PR